public/app/features/dashboard/state/PanelQueryRunner.ts

```diff
@@ -90,6 +90,7 @@ export class PanelQueryRunner {
           processedData = {
             ...processedData,
             series: applyFieldOverrides({
+              autoMinMax: true,
```

Comment:
@ryantxu @dprokop before this was optional, as only some panels where assumed to need it. But now I guess we always have to pass this.